common/src/verify.rs

```diff
@@ -534,28 +534,20 @@ mod test {
         }
     }
 
-    fn generate_empty_transaction_commit(
-        validator_keypair: &[(PublicKey, PrivateKey)],
-        author_index: usize,
-        time: Timestamp,
```

Comment:
Is there a reason for deleting `time` field as well?

Reply:
As mentioned earlier, Simperby protocol does not check the timestamp of the transaction (I also deleted a test that checks the order of timestamp between transactions)